radio/src/Makefile

```diff
@@ -52,7 +52,7 @@ DELAY = 5
 # PCB version
 # Values: 9X, 9X128, 9X2561, 9XR, 9XR128, 9XR2561, GRUVIN9X, MEGA2560, SKY9X, 9XRPRO, AR9X, TARANIS
 # Old values STD and STD128 are mapped to 9X and 9X128 for compatibility
-PCB = 9X
+PCB = 9XRPRO
```

Comment:
Sorry, that was a misunderstanding on my part of how github does pull requests. I assumed the pull request would only cover the changes in the branch, but not in master. So I added my build config to master, and all the changes for the feature in my branch.